sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/Transforms.java

```diff
@@ -1,134 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Just a suggestion, I would have kept the Transforms class as it explains more about the service transform layer and Utility is rather about widely generic helper functions used in the client.